gatsby-config.js

```diff
@@ -1,7 +1,7 @@
-const languages = require('./languages');
+const languages = require("./languages")
```

Comment:
single quotes are preferred these days, and are the default Prettier config, for example. Also, my preference is always terminating with a semicolon, but that's up to you :)